.gitignore

```diff
@@ -7,3 +7,4 @@
 .*.sw*
 /build
 .idea
+.vscode
```

Comment:
You accidentally removed the newline at the end of the end of the file

Reply:
sorry, fixed :)